di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -412,6 +427,62 @@ public AuthorizeHandler(
                 return null;
             };
 
+    private static void processMitigatedCIs(String userId, QueryParamsMap queryParamsMap)
+            throws CriStubException {
+        if (CredentialIssuerConfig.isEnabled(
+                CredentialIssuerConfig.CRI_MITIGATION_ENABLED, "false")) {
+            String mitigatedCIsString =
+                    queryParamsMap
+                            .value(CredentialIssuerConfig.MITIGATED_CONTRAINDICATORS_PARAM)
+                            .replaceAll("\\s", "");
+
+            if (!mitigatedCIsString.isEmpty()) {
+                LOGGER.info("Processing mitigated CI's");
+                String baseStubManagedPostUrl =
+                        queryParamsMap.value(
+                                CredentialIssuerConfig.BASE_STUB_MANAGED_POST_URL_PARAM);
+                String stubManagementApiKey =
+                        queryParamsMap.value(CredentialIssuerConfig.STUB_MANAGEMENT_API_KEY_PARAM);
+                List<String> mitigatedCiList =
+                        Stream.of(mitigatedCIsString.split(",", -1)).collect(Collectors.toList());
+                String postUrlTemplate = "/user/%s/mitigations/%s";
+                for (String ciCode : mitigatedCiList) {
+                    String postUrl =
+                            baseStubManagedPostUrl + String.format(postUrlTemplate, userId, ciCode);
+                    LOGGER.info("Managed cimit stub postUrl:{}", postUrl);
+                    try {
+                        HttpRequest request =
+                                HttpRequest.newBuilder()
+                                        .uri(new URI(postUrl))
+                                        .header("Content-Type", "application/json")
+                                        .header("x-api-key", stubManagementApiKey)
+                                        .POST(
+                                                HttpRequest.BodyPublishers.ofString(
+                                                        "{\"mitigations\": [\"M01 \"]}"))
+                                        .build();
+                        HttpResponse response =
+                                HttpClient.newBuilder()
+                                        .proxy(ProxySelector.getDefault())
+                                        .build()
+                                        .send(request, HttpResponse.BodyHandlers.ofString());
+                        LOGGER.info(
```

Comment:
In case of any exception while posting req it'll redirect to /page/pyi-technical meg-We cannot prove your identity right now.
And log containing detail of post req and error details.